CHANGELOG.md

```diff
@@ -138,6 +138,8 @@ v1.12.0-rc.0
 
 ### Bugfixes
 
+- Fix Docker log corruption for multiplexed long lines. (@axd1x8a)
```

Comment:
Changelog likely needs to be manually updated from main to hit the unreleased section.